.ci.yaml

```diff
@@ -70,7 +70,7 @@ platform_properties:
       os: Mac-10.15
       device_type: none
       mac_model: Macmini8,1
-      xcode: 12a7209
+      xcode: 12c33
```

Comment:
Have we manually tried the mac builds work properly with the new version of xcode?

Reply:
I'd also recommend a soft transition with two named caches. Replacing the cache directly will force a transition period where the caches will be moving back and forward between versions.

Reply:
I naively thought framework and engine used the same xcode, so I didn't validate. This resets it to what it is in starlark

https://cs.opensource.google/flutter/infra/+/main:config/devicelab_config.star;l=18;drc=323e41a1f63df6be76b607290a23bb1fc676de8e
https://cs.opensource.google/flutter/infra/+/main:config/framework_config.star;l=17;drc=323e41a1f63df6be76b607290a23bb1fc676de8e

Reply:
Ahh I see, is this a revert?

Reply:
+1 to using new caches when changing dep versions (I'm writing up how to bringup new versions with ci.yaml, where we'll version the cache paths)

Since the damage is already done (set the devicelab tests to 12a7209, reverted back to 12c33), keeping the caches the same here will have no effect